build.gradle

```diff
@@ -24,22 +24,264 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
+//use new plugins method
+plugins {
+    id 'net.saliman.cobertura' version '3.0.0' apply false
+    id 'org.ajoberstar.git-publish' version '3.0.0' apply false
+    id 'nebula.release' version '15.2.0' apply false
+}
+
 description = 'EMC Object Client for Java - provides REST access to object data on EMC platforms using the Atmos and S3 APIs.'
 
 ext.githubProjectName = 'ecs-object-client-java'
 
-buildscript {
-    ext.commonBuildVersion = '1.6'
-    ext.commonBuildDir = "https://raw.githubusercontent.com/EMCECS/ecs-common-build/v$commonBuildVersion"
-    apply from: "$commonBuildDir/ecs-publish.buildscript.gradle", to: buildscript
-}
-
 allprojects {
-    apply from: "$commonBuildDir/ecs-publish.gradle"
+    group 'com.emc.ecs'
+
+    if (!hasProperty('githubProjectName')) ext.githubProjectName = project.name
+    if (!hasProperty('scmUrl')) ext.scmUrl = "https://github.com/EMCECS/${githubProjectName}"
+    if (!hasProperty('scmConnection')) ext.scmConnection = "scm:git@github.com:EMCECS/${githubProjectName}.git"
+    if (!hasProperty('githubUrl')) ext.githubUrl = "https://github.com/EMCECS/${githubProjectName}.git"
+    ext.aggregatedDocsDir = "$buildDir/aggregatedDocs"
+
+    if (!hasProperty('licenseName')) ext.licenseName = 'The BSD 3-Clause License'
+    if (!hasProperty('licenseUrl')) ext.licenseUrl = 'http://opensource.org/licenses/BSD-3-Clause'
+
+    apply plugin: 'idea'
+    apply plugin: 'eclipse'
+    apply plugin: 'java'
+    apply plugin: 'net.saliman.cobertura'
+    apply plugin: 'distribution'
+    apply plugin: 'signing'
+    apply plugin: 'maven'
+
+    defaultTasks 'distZip'
+
+    def isRootProject = (project == rootProject)
```

Comment:
The rootProject variable is also a common-build construct.  We know this is the root project, so there is no need for this variable.